google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -772,7 +818,7 @@ public void close() {
         if (state != SessionState.CLOSING) {
           state = SessionState.AVAILABLE;
         }
-        releaseSession(this);
+        releaseSession(this, Position.FIRST);
```

Comment:
could u tell me when this action will be performed ?

Should we release the (least recently used) LRU index ?

Reply:
This is called when a pooled session is returned to the session pool (`PooledSession#close()` does not close the session, but checks a session back into the pool). This is therefore the most recently used session, assuming that the client actually used the session while it was checked out, and hence is added to the head of the pool.